selectrum.el

```diff
@@ -169,7 +169,9 @@ strings."
     ([remap end-of-buffer]       . selectrum-goto-end)
     ([remap kill-ring-save]      . selectrum-kill-ring-save)
     ("C-j"                       . selectrum-submit-exact-input)
-    ("TAB"                       . selectrum-insert-current-candidate))
+    ("TAB"                       . selectrum-insert-current-candidate)
+    ("M-p"                       . selectrum-previous-history-element)
+    ("M-n"                       . selectrum-next-history-element))
```

Comment:
These should be `[remap previous-history-element]` and `[remap next-history-element]`, right?

Reply:
Those commands are bound in `minibuffer-local-map`. Selectrum creates its own keymap so this would not work. We could join the keymap with `minibuffer-local-map` but than we need to make sure there are no conflicting bindings in that map which don't work with selectrum.

Reply:
This can be fixed by including `(set-keymap-parent keymap minibuffer-local-map)` during keymap construction, as in CTRLF. I didn't realize this wasn't done already.

Reply:
Currently the remappings in `selectrum-minibuffer-bindings` all remap from global commands but when using  `minibuffer-local-map` as parent those remapping won't work anymore. The remappings need to be adjusted to remap the existing bindings in `minibuffer-local-map`

Reply:
This is done in https://github.com/raxod502/selectrum/commit/551b119075160cca38017491ba694c89d351983a so we should be able to use `[remap ...]` now.